packages/hd/src/keychain.ts

```diff
@@ -1,6 +1,7 @@
-import crypto from "crypto";
-import { ec as EC } from "elliptic";
+/* eslint-disable @typescript-eslint/no-magic-numbers */
 import BN from "bn.js";
+import { ec as EC } from "elliptic";
```

Comment:
can the library 'elliptic' be replaced with the `noble/curve` or `noble/secp256k1`?

Reply:
Yes. This can be done. 
1. Replace node crypto.
2. Replace `Buffer` with `Uint8Array`
3. Optimize bundle package size.    `elliptic` will be replaced in this step.